foundation/foundation-mda/src/main/resources/templates/deployment/argocd/v2/spark.infrastructure.yaml.vm

```diff
@@ -6,6 +6,15 @@ metadata:
   finalizers:
     - resources-finalizer.argocd.argoproj.io
 spec:
+  ignoreDifferences:
+    - group: ""
+      kind: ConfigMap
+      jsonPointers:
+        - /data/metastore-site.xml
+    - group: apps
+      kind: StatefulSet
+      jsonPointers:
+        - /spec/template/spec/containers/0/env/2/value
```

Comment:
Q: Which env is this referring to?

Reply:
This refers to environment variable 
![Screenshot 2024-12-26 at 5 01 48 PM](https://github.com/user-attachments/assets/1d3f1374-230c-40cb-908a-ffe8bc9982f2)